docs/tutorial.md

```diff
@@ -662,10 +662,10 @@ int func()
 Let's compile it and see what it looks like.   Here we compile with `-g`
 to include source line info:
 
-> Note: Replace `.\eBPF-for-Windows.0.6.0\build\native\include\` with the path to the ebpf-for-windows nuget `./include` directory (should be in the directory where you ran `nuget install eBPF-for-Windows`). This is the first time we included header files while building so we need to use the Windows eBPF headers which we get from the nuget package.
```

Comment:
Any way to refactor this file so you don't have to constantly update it for each version? Seems very fragile...

Reply:
Should be outside the scope of this PR in my view.

Reply:
Agree, and I already had this in the next doc revamp.